wrangler-core/src/main/java/io/cdap/directives/validation/conformers/JsonConformer.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright © 2019 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+package io.cdap.directives.validation.conformers;
+
+import com.google.gson.JsonObject;
+import io.cdap.directives.validation.ConformanceIssue;
+
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.ValidationException;
+import org.everit.json.schema.Validator;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/** Validates using JSON Validator. */
+public class JsonConformer implements Conformer<JsonObject> {
+  public static final String SCHEMA_FORMAT = "json";
+
+  private final InputStream schemaStream;
+  private Schema schema;
+
+  private JsonConformer(InputStream stream) {
+    this.schemaStream = stream;
+  }
+
+  @Override
+  public void initialize() {
+    JSONObject schemaJson = new JSONObject(new JSONTokener(new InputStreamReader(schemaStream)));
```

Comment:
It seems odd that none of these methods throw an exception. It means the library swallows any IOException it gets? Or does it re-throw as a runtime exception? 

Is the JSONTokener closing the reader?

Reply:
They do rethrow as runtime. Added try-with-resources for this anyways.